templates/header/topbar_begin.html

```diff
@@ -9,8 +9,11 @@
 <div class="nav-container">
     <div class="container">
         <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
-            <form action="/releases/search" method="GET"
-                class="landing-search-form-nav {% if is_latest_version is defined and not is_latest_version %}not-latest{% endif %}">
+            <form action="/releases/search"
+                  method="GET"
+                  class="landing-search-form-nav {%
+                  if is_latest_version is defined and not is_latest_version %}not-latest{% endif
+                  %} {% if metadata.yanked %}yanked{% endif %}">
```

Comment:
Is this case possible? Can we see the docs of a yanked crate? :o

Reply:
Yes, since you can still depend on yanked crates and use them, we still allow accessing the docs.

Reply:
I didn't know you could still use yanked crates. TIL.